### PR TITLE
Alert cleanup

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

File: selfdrive/controls/lib/events.py [hunk 1]
```diff
@@ -588,27 +581,27 @@ def wrong_car_mode_alert(CP: car.CarParams, sm: messaging.SubMaster, metric: boo
   EventName.calibrationInvalid: {
     ET.PERMANENT: Alert(
       "Calibration Invalid",
-      "Reposition Device and Recalibrate",
+      "Remount Device and Recalibrate",
       AlertStatus.normal, AlertSize.mid,
       Priority.LOWER, VisualAlert.none, AudibleAlert.none, 0., 0., .2),
-    ET.SOFT_DISABLE: SoftDisableAlert("Calibration Invalid: Reposition Device & Recalibrate"),
-    ET.NO_ENTRY: NoEntryAlert("Calibration Invalid: Reposition Device & Recalibrate"),
+    ET.SOFT_DISABLE: SoftDisableAlert("Calibration Invalid: Remount Device & Recalibrate"),
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/events.py:587]: @HaraldSchafer with the new calibration can we just make this "Calibration Invalid: Remount Device"?

Comment (HaraldSchafer) [reply selfdrive/controls/lib/events.py:587]: Hmmm, if they remount with less than 2deg difference it won't trigger a recalibration. So not sure.

Comment (adeebshihadeh) [reply selfdrive/controls/lib/events.py:589]: let's leave it then

Comment (VirtuallyChris) [reply selfdrive/controls/lib/events.py:595]: I think "Calibration Invalid: Reposition Mount" might work here

**Pull request merged**